src/Session.php

```
@@ -480,6 +480,10 @@ public function getCacheLimiter()
      */
     public function setCookieParams(array $params)
     {
+        if ($this->isStarted()) {
+            return;
```

Do you think we should throw an exception and raise an error than suppressing it

---

If you need this return, it is likely that you are doing `setCookieParams` in the wrong place, and surely an exception is better than ignoring it.

---

I think so.

---

@harikt Is the exception name something like `SessionAlreadyStartedExceptionton`?

---

I think we should follow the Aura.Router convention. No Exception at end. Rest looks good to me.